package.json

```diff
@@ -38,30 +38,31 @@
         "continuous integration"
     ],
     "dependencies": {
-        "@types/promise.prototype.finally": "^2.0.2",
-        "@types/i18n": "^0.8.3",
-        "ava": "^0.25.0",
-        "googleapis": "^32.0.0",
-        "google-auth-library": "^1.5.0",
-        "google-proto-files": "^0.16.0",
-        "google-protobuf": "^3.3.0",
-        "grpc": "^1.12.2",
+        "@types/i18n": "^0.8.6",
+        "@types/promise.prototype.finally": "^2.0.3",
+        "ava": "^2.4.0",
```

Comment:
If we're updating these deps, we will need to also update the supported versions in the `engines.node` field.

Reply:
Does this change also affect the `target` on tsconfig.json?